oshi-core/src/main/java/oshi/software/os/mac/MacOperatingSystem.java

```diff
@@ -393,4 +396,92 @@ public NetworkParams getNetworkParams() {
         return new MacNetworkParams();
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public OSService[] getServices() {
+        //Sort by PID
+        OSProcess[] process =  getChildProcesses(1, 0, OperatingSystem.ProcessSort.PID);
+        //Get Directories
+        File ua = new File("~/Library/LaunchAgents");
```

Comment:
The `~` for this path is a relative path, that doesn't work from inside a Java application.  You need to replace the tilde with `System.getProperty("user.home")`